src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java

```diff
@@ -53,12 +57,32 @@
             ChatColor.LIGHT_PURPLE,
             ChatColor.YELLOW);
 
+    public static final List<ChatColor> CHAT_RAINBOW = Arrays.asList(
```

Comment:
I see no valid justification to duplicate code in this way. There is no logical explication that I have read here that tells me why we could not just use the same pool, as all you've done is changed the order...